content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -230,6 +230,10 @@ You can omit the `--config-repo` option if the CI is checked out to the configur
 
 If the command has been correctly run, deployments contain Git metadata from the application repository instead of the configuration repository. Also, the deployment executions view now contains a new **Pipeline** tab representing the related CI pipeline trace.
 
+## Visualize deployments in Datadog
+
+The [**Deployments**][6] and [**Executions**][7] pages populate with data after a deployment is executed. For more information, see [Search and Manage][9] and [CD Visibility Explorer][10].
```

Comment:
```suggestion
The [**Deployments**][6] and [**Executions**][7] pages populate with data after a deployment has finished. For more information, see [Search and Manage][9] and [CD Visibility Explorer][10].
```

I think that executed is too broad, it does not say that data comes only when the deployment has finished.